tests/components/element-editor/basic-details/basic-details.spec.tsx

```diff
@@ -151,6 +151,7 @@ const MOCK_FOCUSED_ELEMENT = {
         id: "mockElement",
         required: false,
         label: ELEMENT_BUTTON_LABELS[EElementType.EMAIL],
+        columns: { desktop: 12, tablet: 8, mobile: 4 } as IColumns,
```

Comment:
update for the rest of your changes too

```suggestion
        columns: { desktop: 12, tablet: 8, mobile: 4 } as const,
```

Reply:
update other instances of this too?